doc/flame/effects.md

```diff
@@ -376,6 +378,9 @@ uses multiple paints, the effect can target any individual color using the `pain
 final effect = OpacityEffect.to(
   0.2,
   EffectController(duration: 0.75),
+  target: component.opacityProviderOfList(
```

Comment:
Should we mention and give an example of passing a target and not passing a target since this is optional?

Reply:
Added this:

![OpacityEffectTargetDemo](https://user-images.githubusercontent.com/33748002/197253826-8571d2ef-b51a-4002-8611-3e333668205b.gif)